static/app/components/events/aiAutofix/fixResult.tsx

```diff
@@ -0,0 +1,146 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import {AutofixData} from 'sentry/components/events/aiAutofix/types';
+import Anchor from 'sentry/components/links/anchor';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
+import PanelHeader from 'sentry/components/panels/panelHeader';
+import {IconOpen} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+import {ExperimentalFeatureBadge} from '../aiSuggestedSolution/experimentalFeatureBadge';
+import {SuggestionLoaderMessage} from '../aiSuggestedSolution/suggestionLoaderMessage';
+
+type Props = {
+  autofixData: AutofixData;
+  onRetry: () => void;
+};
+
+const makeGithubRepoUrl = (repoName: string) => {
+  return `https://github.com/${repoName}/`;
+};
+
+export function FixResult({autofixData, onRetry}: Props) {
+  const isLoading = autofixData.status === 'PROCESSING';
+  const hasNoFix = autofixData.status === 'COMPLETED' && !autofixData.fix;
+  const hasError = autofixData.status === 'ERROR';
+
+  return (
+    <Panel>
+      <Header>
+        <Title>
+          {t('AI Autofix')}
+          <ExperimentalFeatureBadge />
+        </Title>
+      </Header>
+      <PanelBody>
+        {isLoading ? (
+          <LoaderWrapper>
+            <SmallerAiLoader className="ai-suggestion-wheel-of-fortune" />
+            <SuggestionLoaderMessage />
+          </LoaderWrapper>
+        ) : hasError ? (
+          <Content>
+            <PreviewContent>
+              {autofixData.errorMessage ? (
+                <Fragment>
+                  <PrefixText>Something went wrong:</PrefixText>
+                  {autofixData.errorMessage && <span>{autofixData.errorMessage}</span>}
+                </Fragment>
+              ) : (
+                <span>Something went wrong.</span>
+              )}
+            </PreviewContent>
+            <Button size="xs" onClick={onRetry}>
+              {t('Try Again')}
+            </Button>
+          </Content>
+        ) : hasNoFix ? (
+          <Content>
+            <PreviewContent>
+              <span>Could not find a fix.</span>
+            </PreviewContent>
+            <Button size="xs" onClick={onRetry}>
+              {t('Try Again')}
+            </Button>
+          </Content>
+        ) : (
+          <Content>
+            <PreviewContent>
+              <PrefixText>
+                Pull request #63088 created in{' '}
```

Comment:
Is this meant to be hard-coded or should it use the autofix data? Same with the repo name below